test/ui-test/src/screens/StatusAccountsScreen.py

```diff
@@ -38,8 +35,11 @@ def select_account(self, account):
     def __find_account(self, account):
         found = False
         account_obj = None
-        for index in range(self.__accountsList.count):
-            a = self.__accountsList.itemAtIndex(index)
+        __is_loaded = False
+        __accountsList = None
+        [__is_loaded, __accountsList] = is_loaded_visible_and_enabled(SAccountsComponents.ACCOUNTS_POPUP.value)
```

Comment:
I'm wondering if it is this redundant bc when in the constructor you are already checking the screen is loaded and visible